spec/index.md

```diff
@@ -335,163 +325,131 @@ Each dimension is represented by a key in the `dimensions` object. The object ha
 }
 ```
 
-## Examples
-
-{% assign sorted_pages = site.pages | sort:"order" %}
-{% for page in sorted_pages %}
-  {% if page.category == 'example' %}
-  * [{{ page.title }}]({{ page.url | remove: 'index.html' }}) 
-  {% endif %}
-{% endfor %}
-
-# Content
```

Comment:
Ok, I thought about this a bit. Here's my thinking. "Form vs content" is, to me, an unnecessarily abstract way of explaining what this section actually covers. Virtually everything in it can be expressed more concretely, and simply, as restrictions and guidance on dimensions, that depend on the dimension types chosen.

Put differently: everything that you described as "content" can be expressed as "structure", and that's a good thing.

(I removed all the introductory text because essentially it didn't add anything to the spec. It had the feel of "Here's us thinking out loud about why we're doing this", without offering any concrete guidance or helping the reader.)